third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -108,14 +108,20 @@ impl TestConfig {
         // Get path to allow path-specific test configuration
         let path = path.to_string_lossy();
 
-        // turn on simplifier unless doing no-simplifier or variable-coalescing tests.
+        // Paths for which we do not run the AST simplifier.
+        let no_simplifier_paths = vec![
+            "/no-simplifier/",
+            "/variable-coalescing/",
+            "/default-sbc-optimize/",
+        ];
+        // Turn on simplifier unless in one of the above paths.
         if path.contains("/simplifier-elimination/") {
             env_pipeline.add("simplifier", |env: &mut GlobalEnv| {
                 ast_simplifier::run_simplifier(
                     env, true, // Code elimination
                 )
             });
-        } else if !(path.contains("/no-simplifier/") || path.contains("variable-coalescing")) {
+        } else if !(no_simplifier_paths.into_iter().any(|p| path.contains(p))) {
```

Comment:
Lets land this, but please be prepared we will do this entirely differently, I think (inspired by how the Prover does it):

- Every phase as an experiment to control it, when can be specified as 'PHASE[=on|off]' 
- Those experiments are defined via a data structure with default and inheritance mechanism (so that one flag like `OPTIMIZE` can control a group of flags)
- In the test suite, we probably go a declarative way where we define 'test configurations'. 
- Exp files live side-by-side for each configured configuration. A configuration can require the exp file to be shared with another configuration, or has is separate exp file.